### PR TITLE
[TR_184] Coming Soon Modal

### PR DESCRIPTION
**Pull request by yiyu-tian**
### [Trello Card](trello.com/LINK-TO-TRELLO-CARD)
https://trello.com/c/il7guXEN/184-coming-soon-modal

#### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
This task builds out a modal to announce a feature is unavailable and is coming soon.

#### What is the current behavior? (You can also link to an open issue here)
NA

#### What is the new behavior? (if this is a feature change)
NA

#### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)
No.

#### Images (before/ after screenshots, interaction GIFs, ...)
image

#### how to test:
In the "login screen" test code, change the “LinkButton" part into the following:

`<LinkButton
    text="alert Modal"
    onPress={() => {
     updateAlert(
      {
       type: "coming soon",
       dismissable: false,
      },
     );
    }}
/>`

---

### CODE REVIEW

File: src/elements/TheAlertModal/TheAlertModal.tsx [hunk 1]
```diff
@@ -15,48 +16,90 @@ import styles from './TheAlertModal.styles';
...
+				open={alertObj !== undefined}
+				onDismiss={handleDismiss}
+			>
+				<View style={styles.body}>
+					<Icon name="smile" size={75} />
+					<View style={{
+						padding: 5,
+						flexGrow: 1,
+						alignItems: 'center',
+						justifyContent: 'center',
```
Comment (emarti43) [inline src/elements/TheAlertModal/TheAlertModal.tsx:80]: It would be better to refactor this into a new component called `ComingSoonModal` for readability

**Review by emarti43 — CHANGES_REQUESTED** [review]
Everything thing else looks good 👍.  Just need to refactor a bit.

File: src/elements/TheAlertModal/TheAlertModal.tsx (hunk 1 above)
Comment (yiyu-tian) [reply src/elements/TheAlertModal/TheAlertModal.tsx:80]: Got it. That's a good suggestion. I will modify my code.

**Comment by duskcloudxu** [comment]
closing this since the same feature has already been implemented and merged by @bahayg

**Pull request closed without merging**